api/destination.go

```diff
@@ -30,7 +30,7 @@ type DestinationConnection struct {
 
 func (r DestinationConnection) ValidationRules() []validate.ValidationRule {
 	return []validate.ValidationRule{
-		validate.Required("url", r.URL),
+		validate.Required("ca", r.CA),
```

Comment:
Could this requirement break old connectors?

Reply:
No, current connection already consistently pass Connection.CA so this doesn't really do anything. Alternatively, I can remove the validation rules entirely.